### ISSUE TITLE
Make tag width configurable

### ISSUE THREAD

**Issue by rzblue** [PhotonVision/photonvision#910]
Currently tag width is hardcoded to 6". This should be made configurable.

**Comment by mcm001** [comment]
This is actually somewhat difficult to do in our code. Can we instead do 2023_6in and 2024_6.5in models?

**Comment by rzblue** [comment]
I suppose that would be fine. Is this because of the need to have the target model to draw on the image?

**Comment by mcm001** [comment]
Yeah, and right now, target model is saved as an enum. Having a "custom" thing isn't possible to encode in an enum really if we just send the ordinal value/name to the frontend either.

It's a dumb requirement to make everything an enum instance, though. I would be open to just Not Doing That

**Comment by rzblue** [comment]
Right now the tag width is a hardcoded value in `AprilTagPipeline` (https://github.com/photonvision/photonvision/blob/master/photon-core/src/main/java/org/photonvision/vision/pipeline/AprilTagPipeline.java#L72)

I made some quick and dirty changes to add the tag width to the apriltag settings, haven't tested it yet though. Of course, the boxes drawn on the image will be incorrect.
